tsconfig.build.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file will now determine what is being built and included in `lib`. I removed declaration files, source maps, and jsx preservation.